src/comment-replies-cache/index.js

```diff
@@ -80,13 +80,16 @@ function cleanupRepliesCache() {
   });
 }
 
-var LocalStorageMixin = {
+export const LocalStorageMixin = {
   localStorage: {
     getItem: getItem,
     setItem: setItem,
     removeItem: removeItem,
   },
 };
+export const cleanup = cleanupRepliesCache;
 
-module.exports.cleanup = cleanupRepliesCache;
-module.exports.LocalStorageMixin = LocalStorageMixin;
+export default {
+  cleanup,
+  LocalStorageMixin,
+};
```

Comment:
Whats the reason to export them individually and as a `default`ed object?

Reply:
Its so that I don't need to chase down how the consumers are using it. Definitely not something that should be done going forward